### ISSUE TITLE
Add Safari 15.6

### ISSUE THREAD

**Issue by ahmetgeymen** [browscap/browscap#2694]
# User agent(s)

```
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.6 Safari/605.1.15'
'Mozilla/5.0 (iPhone; CPU iPhone OS 15_6 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.6 Mobile/15E148 Safari/604.1'
'Mozilla/5.0 (iPad; CPU OS 15_6 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.6 Mobile/15E148 Safari/604.1'
```

# Browscap version tested on

6001000 (or 6.1.0)

# Description / Further information

Safari 15.6 on macOS Monterey, Safari on iOS 15.6 and iPadOS 15.6 is not recognized on browscap. It shows as version 0.0

**Comment by ahmetgeymen** [comment]
I've opened a PR for this issue.
https://github.com/browscap/browscap/pull/2695

**Comment by asgrim** [comment]
Fixed in #2695
